src/modules/login/ResetPassword.tsx

```diff
@@ -74,30 +73,32 @@ function ResetPassword() {
           onSubmit={onSubmit}
         >
           {(formState) => (
-            <Form className={classes.form}>
-              <XTextInput
-                type="password"
-                name="password"
-                label="Password"
-                autoComplete="off"
-                margin="normal"
-                onChangeCapture={handleChange}
-                onKeyUp={validation}
-              />
-              <Button
-                type="submit"
-                fullWidth
-                variant="contained"
-                color="primary"
-                className={classes.submit}
-                disabled={formState.isSubmitting}
-              >
-                Update Password
-              </Button>
-            </Form>
+            <>
+              <Form className={classes.form}>
+                <XTextInput
+                  type="password"
+                  name="password"
+                  label="Password"
+                  autoComplete="off"
+                  margin="normal"
+                  onChangeCapture={handleChange}
+                  onKeyUp={validation}
+                />
+                <Button
+                  type="submit"
+                  fullWidth
+                  variant="contained"
+                  color="primary"
+                  className={classes.submit}
+                  disabled={formState.isSubmitting}
+                >
+                  Update Password
+                </Button>
+              </Form>
+              <Typography variant="body2" style={{color: isLength ? "green" : "red"}}>Password must be 8 characters long</Typography>
```

Comment:
You really do not need this, refer to [form field validation](https://material-ui.com/components/text-fields/#validation)